DevelopGitHub/GitHubExtension/GitHubExtension.Tools.LocalizationTool/Translate/Translator.cs

```diff
@@ -0,0 +1,159 @@
+using System;
+using System.Collections.ObjectModel;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Threading.Tasks;
+using GitHubExtension.LocalizationTool.Model;
+
+using Newtonsoft.Json.Linq;
+
+namespace GitHubExtension.LocalizationTool.Translate
+{
+    public class Translator
+    {
+        public const string YandexTranslateApiUrl =
+            "https://translate.yandex.net/api/v1.5/tr.json/translate?" +
+            "key=trnsl.1.1.20160322T103501Z.10b2b142f2f8bf7f.66c4f9f75232ede5cb9d8cc5ce17df5fd1d02d32&" +
+            "lang=";
+
+        private const string UsLang = "us";
```

Comment:
Can you explain what means EnLang? You don't want using more readable Language word? :) Feel free to use more symbols it's very cheeper for us.